vtex/faststore
# Feat: Adds `RegionBar` component

## What's the purpose of this pull request?

- Adds `RegionBar` component to `faststore/components`
- Refactor `RegionBar` component on `core`
- Adds `RegionBar` doc page

![image](https://user-images.githubusercontent.com/3356699/228696377-3c297716-fd73-4ee8-9a61-521d27c90239.png)

![image](https://user-images.githubusercontent.com/3356699/228696424-2bc21635-d613-4710-84ad-9102457acacb.png)

- [x] Renames Regionalization to Region
Ps: I've only left `RegionalizationModalContent` intentionally. We will work on this component soon.



## How to test it?

1. Run `yarn dev` on core package
- Region feature should be working as before. You can try open the Region Modal through the RegionBar or RegionButton.

2. Check the `RegionBar` doc page using the preview.3. 

### Starters Deploy Preview

[WIP]
